staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_discovery_controller.go

```diff
@@ -74,24 +80,27 @@ func NewDiscoveryController(crdInformer informers.CustomResourceDefinitionInform
 	return c
 }
 
-func (c *DiscoveryController) sync(version schema.GroupVersion) error {
+func (c *DiscoveryController) sync(gvt GroupVersionTenant) error {
 
 	apiVersionsForDiscovery := []metav1.GroupVersionForDiscovery{}
 	apiResourcesForDiscovery := []metav1.APIResource{}
 	versionsForDiscoveryMap := map[metav1.GroupVersion]bool{}
 
-	crds, err := c.crdLister.List(labels.Everything())
+	groupVersion := schema.GroupVersion{Group: gvt.group, Version: gvt.version}
+
+	crds, err := c.crdLister.CustomResourceDefinitionsWithMultiTenancy(gvt.tenant).List(labels.Everything())
```

Comment:
nit: CustomResourceDefinitionsWithMultiTenancy --> CRDWithMultiTenancy

Reply:
Cannot do it. :-(

CustomResourceDefinitionsWithMultiTenancy is a name auto-generated by client-go.